### DISCUSSION TITLE
Handling formEl.submit()

### DISCUSSION THREAD

**Discussion by alexandremjacques** [unpoly/unpoly#315] [General]
Hi.

I have a use case where I need to update filters based on user selection on various `<select>` fields. My problem is that, apparently, if I place an `onchange="this.form.submit()"` on the `<select>`, the submit event is not triggered. I guess that breaks `up-submit`.

Does anybody have seen this? Any ideas?

TIA

**Comment by triskweline** [comment]
If you'd like to submit a form through Unpoly, check out the [`up.submit()`](https://unpoly.com/up.submit) function.

Also your use case may be a good fit for the [`[up-autosubmit]`](https://unpoly.com/form-up-autosubmit) attribute.

**Reply by alexandremjacques** [reply]
Thanks. Worked like a charm.
